shapely/geometry/base.py

```diff
@@ -721,23 +721,22 @@ def dwithin(self, other, distance):
         """
         return _maybe_unpack(shapely.dwithin(self, other, distance))
 
-    def equals_exact(self, other, tolerance):
-        """True if geometries are equal to within a specified
-        tolerance.
+    def equals_exact(self, other, tolerance=0.0, normalize=False):
+        """Returns True if the objects have the same sequence of coordinates
```

Comment:
Should match the first part of the docstring for `equals_exact()` function.

Reply:
sorry, I did not get that, could you be more explicit?

Reply:
I think that he meant that the docstring for `equals_exact` in `base.py` should be the same as the first part (i.e. first few lines) of `equals_exact` in `predicates.py` (which holds the full doc of this function)

Reply:
Ah, right!
If it's fine with you an until @idanmiara deals with the duplication issue in the doc, I'd propose to copy the text from manual.rst to both ``base.py`` and ``predicates.py`` if that's fine with both of you.

Reply:
@tfardet there is no need to duplicate the docstring in `manual.rst`.
The manual will also contain the docs for this function as it pulls is from `predicates.py`, see https://shapely.readthedocs.io/en/stable/predicates.html for some examples.

A single copy of the full docstring should be where the function is implemented, which is in `predicates.py`.
the convenience wrapper in `base.py` should only contain the first few lines (i.e. the gist) of the full docstring (from `predicates.py`)

Reply:
In my experience, the manual is the most consulted page, so until it pulls automatically from predicates, I'm not willing to reduce the docstring there ;)

Reply:
> In my experience, the manual is the most consulted page, so until it pulls automatically from predicates, I'm not willing to reduce the docstring there ;)

Please note that that both `manual` and `predicates` are pages of the main shapely docs (see the links on the left):
https://shapely.readthedocs.io/
I'm thinking that since `equals_exact` already is mentioned in the `manual` page as a gist, maybe update the it in the same way as in `base`. 
I think in general we should avoid duplicating docs also because it makes it harder for keeping them in sync when editing, so we only maintain a single copy of the full doc and copy short gists where appropriate.
@brendan-ward what do you think ?

Reply:
IMO the right solution for this is to use

```rst
.. autosummary::

    shapely.predicates.equals_exact
```

or that of ``BaseGeometry.equals_exact`` in ``manual.rst``.

Once that is decided, we can put the main docstring in only one of the two and reuse it directly in manual.rst.
I thought that was what you meant when you said you would work on cleaning up the doc.

Reply:
Sorry for the confusion!  Yes, what I meant was that the authoritative and complete docstring should be on the `equals_exact` function, and just a snippet of it in the `equals_exact` method.  Unfortunately I added a lot of my text suggestions to the entry in the manual not the docstring! (wasn't paying close enough attention to location, only content)  Sorry, my intent was that it would apply to the docstring instead.

For now, having that duplicated into the manual is OK; it would be fine to resolve that in a follow-up PR.  But if using 

```
.. autosummary::

    shapely.predicates.equals_exact
```

is a drop-in change, that is also fine to do here (using function not method).

Reply:
No pb!
Regarding the autosummary, I think that needs to be done properly an consistently throughout the manual, once and for all, with a proper discussion, so I'll leave that for a later PR. For now let's stick with the duplicated entry in the manual ;)